build.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    println!("cargo:rustc-link-lib=dylib=sensors");
```

Comment:
what is the advantage/reason for moving this to the `build.rs` instead of keeping it in the `Cargo.toml`?

Reply:
This allows one to override the keys as using a local Cargo configuration (cf. <https://doc.rust-lang.org/cargo/reference/build-scripts.html#overriding-build-scripts>). IMHO we should accept and merge @Camille019’s PR.